packages/main/src/Badge.js

```diff
@@ -32,6 +32,22 @@ const metadata = {
 			type: String,
 			defaultValue: "1",
 		},
+
+		/**
+		 * Defines if the badge has an icon.
+		 * @private
+		 */
+		__hasIcon: {
```

Comment:
Do we really need two underlines in the beginning. We usually use only 1

Reply:
Done